apps/frontend/src/components/DashBoard.tsx

```diff
@@ -262,6 +255,17 @@ const Dashboard = () => {
     setOpen(false);
   };
 
+  useEffect(() => {
+    if (isTabletOrMobile) {
```

Comment:
Just to understanding when we switch to mobile the drawer is automatically closed.

Reply:
Yes you are right. That is the idea to hide the drawer by default in mobile views and make more space for the pages that need to be shown. The bug here was that you were not able to re-open it if you do the switch from desktop view to mobile which was a bit tricky.

Reply:
Also users normally login with either a mobile/table or desktop. But with the assumption that some users may switch between the two looks fine to fix.